### PR TITLE
[FIX WEBSITE-214] - Add link to Jenkins wiki

### PR DESCRIPTION
**Pull request by michaelmccaskill**
Related to issue [WEBSITE-214]

Summary of this pull request: 

Add link to Jenkins wiki.

I've emphasized the change in the red rectangle.
wiki link

A closeup if it's hard to see
wiki link 2

### CODE REVIEW

**Comment by daniel-beck** [comment]
FYI https://wiki.jenkins-ci.org/display/JENKINS/GitHub+commit+messages

**Comment by michaelmccaskill** [comment]
Should the others then be adjusted, too?

**Comment by daniel-beck** [comment]
Already done so, except one where I'm not positive it's the final solution (see comment in Jira).

File: app/components/PluginDetail.jsx [hunk 1]
```diff
@@ -176,6 +176,11 @@ class PluginDetail extends React.PureComponent {
                 </div>
                 <h5>Labels</h5>
                 {this.getLabels(plugin.labels)}
+                
+                <div className="update-link">
+                <h6>Are you maintaining this plugin?</h6>
+                <p>Visit the <a href={plugin.wiki.url} target="_wiki">Jenkins Plugin Wiki</a> to edit this content.</p>
```
Comment (daniel-beck) [inline app/components/PluginDetail.jsx:188]: Does this get quoted correctly in case of weird wiki URLs?

Comment (michaelmccaskill) [reply app/components/PluginDetail.jsx:188]: I'm fairly sure that it does but I would be happy to verify that. Do you have any particular URLs in mind?

Comment (daniel-beck) [reply app/components/PluginDetail.jsx:188]: It seems all wiki URLs are harmless so can probably leave this as is.

These are the only ones outside `a-zA-Z0-9._+-`:

https://wiki.jenkins-ci.org/display/JENKINS/Centralized+Job%28Re%29Action+Plugin
https://wiki.jenkins-ci.org/display/JENKINS/Compuware+Source+Code+Download+for+Endevor%2C+PDS%2C+and+ISPW+Plugin
https://wiki.jenkins-ci.org/display/JENKINS/Dead+Man%27s+Snitch+Plugin
https://wiki.jenkins-ci.org/display/JENKINS/Hudson+Speaks%21+Plugin
https://wiki.jenkins-ci.org/display/JENKINS/Mobile+App+Distribution+%28MDT%29++Plugin
https://wiki.jenkins-ci.org/display/JENKINS/Plugin+Usage+Plugin+%28Community%29

And plugins with exceptionally weird page titles get a completely different URL that's safe to not quote.

https://wiki.jenkins-ci.org/pages/viewpage.action?pageId=60915753
https://wiki.jenkins-ci.org/pages/viewpage.action?pageId=99910492
https://wiki.jenkins-ci.org/pages/viewpage.action?pageId=99910545

**Comment by daniel-beck** [comment]
Merge conflict.

File: app/components/PluginDetail.jsx (hunk 1 above)
Comment (michaelmccaskill) [reply app/components/PluginDetail.jsx:188]: Well I locally tested all of the supplied links and they properly go to their respective wiki pages.

**Comment by michaelmccaskill** [comment]
I resolved the merge conflict.

**Pull request merged**